src/doc/src/reference/unstable.md

```diff
@@ -1227,31 +1226,6 @@ The supported values for `FREQUENCY` are 'always` and 'never', which control
 whether or not a message is printed out at the end of `cargo build` / `cargo check`.
 
 
-### patch-in-config
```

Comment:
Can you add a section at the bottom of the file that indicates which version this is stabilized in (roughly follow the form of the other "stabilized" sections).

Reply:
Done in https://github.com/rust-lang/cargo/pull/9839/commits/e2777ee6aa0c92ac5893e39a01177053bd86b39f !